### ISSUE TITLE
MongoDB\Driver\Manager not found

### ISSUE THREAD

**Issue by virtueless** [mongodb/mongo-php-library#252]
Hello,

I'm very new to Mongo and I need your help.
I installed mongo on Windows 10. I ran mongod.exe and successfully connected to it with mongo.exe.

However, when I try to connect via PHP using:
`$client = new MongoDB\Client(); // with or without params`
Connection loads indefinetely until Firefox tells me the website is unavailable. I tried using 127.0.0.1 instead of localhost as I could read online, but I have the same result.

None of Apache, PHP or mongo gives me an error or some kind of log.

I tried to look where it was hanging, and I see the Mongo\Driver\Manager class seems to be missing.
As it's not in your repo, I think I misunderstood the whole thing. 

Can you help me with this, please?

**Comment by moderndeveloperllc** [comment]
You may get more people looking at your question at a site such as [StackOverflow](http://stackoverflow.com/search?q=mongo+windows). A few tips in trying to figure out the issue:

1. Can you connect to your mongod instance [via the command line](https://docs.mongodb.com/manual/mongo/)?
2. Can you see that the mongod instance up and running with Task Manager?
3. In your `phpinfo()` output, do you see the mongodb extension installed?
4. Can you connect with [`MongoDB\Driver\Manager`](http://php.net/manual/en/mongodb-driver-manager.construct.php) directly - not using this wrapper library?

**Comment by bjori** [comment]
I think you may have forgotten to install the actual extension itself, and only installed the userspace library.

See: http://mongodb.github.io/mongo-php-library/getting-started/#requirements

**Comment by virtueless** [comment]
Thank you for your replies.

@moderndeveloperllc 
1. Yes.
2. Yes.
3. Yes.
4. No!

@bjori I installed php_mongodb.dll and set it in my php.ini. It's the same as mongodb.so, right?

OK, I can't use MongoDB\DriverManager directly, so it must comes from my extension. But I did installed it. WAMP tells me it's installed and used and phpinfo() shows me the mongodb extension as well.

Edit: I just installed the stable version instead of alpha 1.2 and put it in both php5 and php7 ext folder and now it works.

Thanks, guys! Sorry for this unrelated problem.
